plugins/admissions/models.py

```diff
@@ -270,8 +273,11 @@ class Meta:
         'ROOM'                                  : 'room',
         'BED'                                   : 'bed',
         'TRANS_HIST_TRANSFER_REASON'            : 'transfer_reason',
-        'CREATED_DATE'                          : 'created_datetime',
         'UPDATED_DATE'                          : 'updated_datetime',
+        'TRANS_UPDATED'                         : 'trans_updated_datetime',
```

Comment:
These aren't the fieldnames it'll set them as properties and never save them - imagine we didn't test on test and check data ?